python/tskit/trees.py

```diff
@@ -8777,12 +8871,20 @@ def Fst(
         # around with indexes and samples sets twice.
 
         def fst_func(sample_set_sizes, flattened, indexes, **kwargs):
```

Comment:
Would it be simpler to just duplicate the signature here rather than use the kwargs shortcut? Then you shouldn't need to do any messing around with inspect

Reply:
lol, good idea!